index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>IsaMarvin | Software Engineer</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav>
+        <img src="./images/my-logo.svg" alt="logo" />
```

Comment:
- [OPTIONAL] I think it would be better to use `a` tag for logo 🔗 This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info ✨